thermodynamics/make_component_table.py

```diff
@@ -94,7 +94,7 @@
 outFile.write("# Concerning temperature and pressure ranges, the following parameters have been used:\n")
 outFile.write(f"# min temperature = {minTemp}, max temperature = {maxTemp}, #temperature sampling points = {nTemp}\n")
 outFile.write(f"# min pressure = {minPress}, max pressure = {maxPress}, #pressure sampling points = {nPress}\n#\n")
-outFile.write("# temperature [°C],     pressure [Pa],   density [kg/m3],  viscosity [Pa.s],   enthalpy [J/kg]\n")
+outFile.write("# temperature [°C],     pressure [Pa],   density [kg/m3],  viscosity [Pa.s],   enthalpy [J/kg], thermal_conductivity [W/m.K], cv [J/kg.K], cp [J/kg.K]\n\n")
```

Comment:
Why the second line break? I guess that this might confuse some csv readers?

Reply:
This was a stupid typo. Removed it.